gwt-client-vue/src/main/java/nl/aerius/wui/vue/directives/VectorDirective.java

```diff
@@ -60,8 +61,12 @@ public void inserted(final Element el, final VNodeDirective binding, final VNode
         .forEach(v -> {
           final Attr attr = Js.cast(el.attributes.get(v).cloneNode(true));
 
-          final Node node = el.childNodes.getAt(0);
-          if (node != null && node.hasAttributes()) {
+          final Optional<Node> firstNode = el.childNodes.asList().stream()
```

Comment:
You can take the construction of this `Optional<Node>` and comparison below out of the `forEach` right?

Reply:
Mmmm yeah